runhouse/resources/hardware/launcher_utils.py

```diff
@@ -239,17 +242,46 @@ def up(cls, cluster, verbose: bool = True, force: bool = False):
         data = read_resp_data(resp)
         logger.info("Successfully launched cluster")
         cls._update_from_den_response(cluster=cluster, config=data)
+        cluster.launched_status = "UP"
+
+    @classmethod
+    def check_status(cls, cluster, ping_cluster: bool = False) -> Optional[dict]:
```

Comment:
should this ever be returning anything? also this updates the the cluster object in place with the updated compute property attributes?